blink/src/main/java/com/example/androidthings/simplepio/BlinkActivity.java

```diff
@@ -84,8 +85,14 @@ public void run() {
             }
             try {
                 // Toggle the GPIO state
-                mLedGpio.setValue(!mLedGpio.getValue());
-                Log.d(TAG, "State set to " + mLedGpio.getValue());
+                if (!BoardDefaults.isNxpBoard()) {
```

Comment:
+1. This is actually the correct behavior, because the API Javadoc states that the result of getValue is unpredictable when the pin is configured as output.